### ISSUE TITLE
maybe wrong in PadCollate

### ISSUE THREAD

**Issue by ChenHuaYou** [devJWSong/gpt2-chatbot-pytorch#11]
class PadCollate():
    def __init__(self, eos_id):
        self.eos_id = eos_id
        
    def pad_collate(self, batch):
        input_ids, token_type_ids, labels =[], [], []
        for idx, seqs in enumerate(batch):
            input_ids.append(torch.LongTensor(seqs[0]))
            token_type_ids.append(torch.LongTensor(seqs[0]))
            labels.append(torch.LongTensor(seqs[2]))
            
        input_ids = torch.nn.utils.rnn.pad_sequence(input_ids, batch_first=True, padding_value=self.eos_id)
        token_type_ids = torch.nn.utils.rnn.pad_sequence(token_type_ids, batch_first=True, padding_value=self.eos_id)
        labels = torch.nn.utils.rnn.pad_sequence(labels, batch_first=True, padding_value=-100)
    
        return input_ids, token_type_ids, labels

 token_type_ids.append(torch.LongTensor(seqs[0])) should change 0 to 1

**Comment by devjwsong** [comment]
Hi, thank you for notifying me.
I don't know why this stupid error is discovered now.
Maybe I did a mistake when conducting recent update.
Thank you very much.
